### DISCUSSION TITLE
TRRSケーブルで繋いだ子キーボードの接続不良

### DISCUSSION THREAD

**Discussion by mashu0507** [Yowkees/keyball#612] [Q&A]
今週届いたKeyball39を組立、使用中にちょこちょこ左手子キーボードに下記現象が起こります。
#610の方と似ているかもしれませんが、使用中子キーボードがロゴが消え反応しなくなることがあります。

子キーボードにつなげたTRRSケーブルを回転させたり、抜き差しするとロゴが点いたり消えたりするので接触不良だと思います。
TRRSケーブルの両端を入れ替えても同じ現象になります。

![20240613_160506](https://github.com/Yowkees/keyball/assets/172577505/958dbf8b-a329-423c-860e-30a27d90b85a)
![20240613_160538](https://github.com/Yowkees/keyball/assets/172577505/832521bf-191a-4011-8875-7079be7090ab)
![20240613_160555](https://github.com/Yowkees/keyball/assets/172577505/df5c2ec3-166b-43f3-bb59-c6881faa9836)
![20240613_160451](https://github.com/Yowkees/keyball/assets/172577505/75f13628-a92c-4508-b617-b741a401648f)

はんだ付けは問題ないように見えるのですが、、、何卒宜しくお願い致します。

**Comment by Ryo624** [comment]
はんだ付けは問題ないように見えますが念のためTRRSコネクタを温めなおすとどうなりますでしょうか。端子に馴染むように行ってみてください。
また、予備のものがあればTRRSケーブルも別のものにしたときに接続がどうなるか確認していただけますと幸いです。
